web/src/routes/datasets/Datasets.tsx

```diff
@@ -34,9 +44,24 @@ interface DispatchProps {
   resetDatasets: typeof resetDatasets
 }
 
+interface SearchState {
+  searchTerm: string
+}
+
 type DatasetsProps = WithStyles<typeof styles> & StateProps & DispatchProps
 
-class Datasets extends React.Component<DatasetsProps> {
+class Datasets extends React.Component<DatasetsProps, SearchState> {
+  constructor(props: DatasetsProps) {
+    super(props)
+    this.state = {
+      searchTerm: ''
+    }
+  }
+  
+  handleSearch(event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement> ) {
```

Comment:
I think it's only an inputElement here, no need for the search area.

Reply:
@phixMe When I remove HTMLTextAreaElement from the type, it doesn't compile and I get this error msg: `Argument of type 'ChangeEvent<HTMLInputElement | HTMLTextAreaElement>' is not assignable to parameter of type 'ChangeEvent<HTMLInputElement>'.` Can you advise?